src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
+use Symfony\Component\HttpKernel\EventListener\DebugHandlersListener;
+
+return static function (ContainerConfigurator $container) {
+    $kernelDebugServiceId = 'kernel.debug';
```

Comment:
To be consistent with other PR, let's use the value `kernel.debug` directly for both params.